model/config.go

```diff
@@ -992,6 +994,10 @@ func (s *ExperimentalSettings) SetDefaults() {
 	if s.EnableAppBar == nil {
 		s.EnableAppBar = NewBool(false)
 	}
+
+	if s.EnableVoiceMessages == nil {
+		s.EnableVoiceMessages = NewBool(true)
```

Comment:
Is this expected to be default on?

Reply:
I figured that if the user set the feature flag to true, chances are they want this active so it made sense to have the Experimental to true. I'm 0/5 on that and can change to false if you think it's better

Reply:
Not sure myself, @esethna could you remind me of your proposal when it comes to flags?

Reply:
Posted here for visibility and thoughts: https://community.mattermost.com/core/pl/wopjkwfaxpdt9cq488j4cje5ch